Mathlib/AlgebraicGeometry/EllipticCurve/Projective.lean

```diff
@@ -1163,4 +1176,271 @@ lemma addXYZ_of_Z_ne_zero {P Q : Fin 3 → F} (hP : W.Equation P) (hQ : W.Equati
 
 end Addition
 
+section Negation
+
+/-! ### Negation on point representatives -/
+
+variable (W') in
+/-- The negation of a point representative. -/
+def neg (P : Fin 3 → R) : Fin 3 → R :=
+  ![P x, W'.negY P, P z]
+
+lemma neg_X (P : Fin 3 → R) : W'.neg P x = P x :=
+  rfl
+
+lemma neg_Y (P : Fin 3 → R) : W'.neg P y = W'.negY P :=
+  rfl
+
+lemma neg_Z (P : Fin 3 → R) : W'.neg P z = P z :=
+  rfl
```

Comment:
Should these be `@[simp]` lemmas?

Reply:
Yes, I do think these can be `@[simp]` lemmas, but I deliberately avoided adding the tags at this stage. There will be at least two more PRs that follow this one, and there won't be any `@[simp]` tags for those either. At this point I have not decided what should be the simp normal form, and I was thinking to add them all at the end once the entire API is clear, so as to avoid simpNF linter errors. Let me know if I'm just overthinking, or if this is a bad idea.

Notice that the Jacobian file also has almost no simp lemmas for the exact same reason.